aviary/docs/getting_started/onboarding_ext_subsystem.ipynb

```diff
@@ -231,16 +232,18 @@
    "source": [
     "prob = av.AviaryProblem()\n",
     "\n",
-    "prob.load_inputs('models/test_aircraft/aircraft_for_bench_FwFm.csv', phase_info)"
+    "prob.load_inputs('models/test_aircraft/aircraft_for_bench_FwFm.csv',\n",
+    "                 phase_info, meta_data=ExtendedMetaData)"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "96ebdf23",
    "metadata": {},
    "source": [
+    "Note that we have set `meta_data` to `ExtendedMetaData` so that we can set battery related variables in the model .csv file.\n",
```

Comment:
Reword: "Since this example contains new variables in the aircraft hierarchy, the metadata for those variables was added to an extended metadata dictionary. We need to pass that into load_inputs so that it can load susbsystem-specific inputs from the csv file.

Reply:
done.